test/quantization/test_quantize_fx.py

```diff
@@ -575,8 +575,8 @@ def forward(self, x):
         original_ref_m.conv2.bias = torch.nn.Parameter(original_m.standalone.conv.bias.detach())
 
         qconfig_dict = {"": default_qconfig}
-        config_name = {"standalone_module_name": ["standalone"]}
-        config_class = {"standalone_module_class": [StandaloneModule]}
+        config_name = {"standalone_module_name": [("standalone", None, None)]}
```

Comment:
Can we also test for selective quantization of inputs/outputs for standalone modules?

Reply:
yes, it's in the next PR

Reply:
Sorry, just saw that